### DISCUSSION TITLE
Icon did not render

### DISCUSSION THREAD

**Discussion by DrJohan** [mcanouil/quarto-iconify#34] [Q&A]
Hi Mcanouil

I've been trying to use your extension for my website. However after installing the extension and trying to used that extension in the _quarto.yml, I'm still couldn't see the icon rendered when I do the local preview. Can you help? This is the [link](https://github.com/DrJohan/quartowebsite/tree/master) to my repo for that website. Thanks

**Comment by mcanouil** [comment]
Could you elaborate a bit more on what and where it does not render exactly?
Are you sure the icon exists?

Note that I am pretty sure the extension works properly as all the icons I use on my website come from it.

- https://mickael.canouil.fr
